modules/statics/src/base.ts

```diff
@@ -711,10 +712,9 @@ export enum UnderlyingAsset {
   'talgo:180447' = 'talgo:180447',
 
   // EOS tokens
-  CHEX = 'CHEX',
-  IQ = 'IQ',
-  EOS_BOX = 'BOX',
-  EOS_USDT = 'USDT',
+  CHEX = 'chex',
+  IQ = 'iq',
+  EOS_BOX = 'eos:box',
```

Comment:
seems like a non-trivial change, are we safe in downstream? Also this seems to introduce a new pattern of asset naming. How do we know this is safe and is generic enough for other assets to use as a pattern?

Reply:
i see we do it for algo, so should be ok. Just need to make sure we are setting this as a pattern.

Reply:
great call out - i am considering to use a chain-agnostic name.
i.e. change `EOS_BOX` to `TOKEN_DEFI_BOX` (https://bloks.io/tokens/BOX-eos-token.defi).

Another example is USDT (Tether), which exists on a bunch of chains. Luckily, they seem to have done well with branding/integrating so that USDT is synonymous with Tether.

Reply:
i think chain specific is fine ... but do we then have teos:box and eos:box?

Reply:
we do not, we use the same underlying asset for eos and teos. we use contract name to differentiate.
Using CHEX as an example:
prod contract name = `chexchexchex`
test contract name = `testtoken111`

```
eosToken('eos:CHEX', 'Chintai', 8, 'chexchexchex', UnderlyingAsset.CHEX, AccountCoin.DEFAULT_FEATURES, '', 'CHEX'),
teosToken('teos:CHEX', 'Chintai', 8, 'testtoken111', UnderlyingAsset.CHEX, AccountCoin.DEFAULT_FEATURES, '', 'CHEX'),
```